.github/workflows/tf-aws-test.yml

```diff
@@ -56,10 +61,15 @@ jobs:
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
           aws-region: ${{ env.AWS_REGION }}
       - name: Execute wing test in matrix directory
+        uses: nick-fields/retry@v2
         env:
           TF_LOG: info
           TF_LOG_PATH: ${{ runner.workspace }}/terraform.log
-        run: wing test -t tf-aws ${{ matrix.test.directory }}/*.w
+        with:
+          max_attempts: 3
+          retry_on: error
+          timeout_minutes: 20
+          command: wing test -t tf-aws ${{ matrix.test.directory }}/*.w
```

Comment:
I kind of feel like maybe we want to add this retry in `wing test`, no?

Reply:
Good point, I'll open an issue!
From what I experienced so far while developing and heavily using this command, it hardly fails for "no reason", and when it fails I want to respond the fastest to save my time (if I ran a whole directory and one file failed, next time I'll run only the failed once, and only after adjusting the code, I hardly run the tests without changing something at my end first), this retry option will mostly be valued for automation when we want to make sure something isn't working just because of "environmental factors" rather a problem in the code, it still be good and make the overall testing time shorter- running over only the failed ones.

Reply:
https://github.com/winglang/wing/issues/3796